src/Query/QueryBuilder.php

```diff
@@ -3,6 +3,7 @@
 namespace Doctrine\DBAL\Query;
 
 use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Driver\ResultStatement;
```

Comment:
This looks like a mistake.

Reply:
Indeed, and it's weird phpcs has nothing to complain about this :thinking: